src/runtime_src/driver/xclng/xrt/user_gem/shim.cpp

```diff
@@ -382,7 +385,7 @@ int xocl::XOCLShim::xclLogMsg(xclDeviceHandle handle, xclLogMsgLevel level, cons
     len++; //To include null terminator
```

Comment:
Never use post decrement unless the previous value is needed.

Reply:
Done